runtimes/eden/src/lib.rs

```diff
@@ -381,6 +384,65 @@ sp_api::impl_runtime_apis! {
 
 			impl frame_system_benchmarking::Config for Runtime {}
 
+			use xcm::latest::prelude::*;
+			use frame_benchmarking::BenchmarkError;
+			use pallet_xcm_benchmarks::generic::Pallet as XcmBenchmarks;
+			impl pallet_xcm_benchmarks::Config for Runtime {
+				type XcmConfig = xcm_config::XcmConfig;
+				type AccountIdConverter = xcm_config::LocationToAccountId;
+				fn valid_destination() -> Result<MultiLocation, BenchmarkError> {
+					Ok(MultiLocation::parent())
+				}
+				fn worst_case_holding() -> MultiAssets {
+				// 1 fungibles can be traded in the worst case
```

Comment:
To me it looks the worst case is holding non fungible assets which are bigger objects and could be potentially more than one asset. However it could be okay to start modelling only a single fungible asset which is NODL.

Reply:
Perhaps create a ticket for readjusting the benchmarks after supporting and testing trading of NFTs.